packages/components/src/form/TagsInput/TagsInput.js

```diff
@@ -1,4 +1,4 @@
-import React, { forwardRef, useMemo, useRef, useState } from 'react';
+import React, { forwardRef, useEffect, useMemo, useRef, useState } from 'react';
```

Comment:
useEffect is not being used :D

```suggestion
import React, { forwardRef, useMemo, useRef, useState } from 'react';
```